CorsixTH/Lua/app.lua

```diff
@@ -627,14 +627,21 @@ function App:loadLevel(level, difficulty, level_name, level_file, level_intro, m
       world = {
         room_built = self.world.room_built,
         campaign_info = self.world.campaign_info,
+        debug_disable_salary_raise = self.world.debug_disable_salary_raise,
         },
       hospital = {
         player_salary = self.ui.hospital.player_salary,
         message_popup = self.ui.hospital.message_popup,
         handyman_popup = self.ui.hospital.handyman_popup,
         hospital_littered = self.ui.hospital.hospital_littered,
         has_seen_pay_rise = self.ui.hospital.has_seen_pay_rise,
-      },
```

Comment:
Actually, the existing dedented closing bracket was the common form, the indented closing bracket doesn't happen (much).